src/chart.zoom.js

```diff
@@ -102,23 +108,36 @@ function zoomTimeScale(scale, zoom, center) {
 	var minDelta = newDiff * min_percent;
 	var maxDelta = newDiff * max_percent;
 
-	options.time.min = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
-	options.time.max = scale.getValueForPixel(scale.getPixelForValue(scale.lastTick) - maxDelta);
+	var newMin = scale.getValueForPixel(scale.getPixelForValue(scale.firstTick) + minDelta);
+	options.time.min = newMin < min_limit ? min_limit : newMin;
```

Comment:
Good to know. Which values can be used for the limits except the momentjs objects?